tests-example/Tiltfile.tests

```diff
@@ -0,0 +1,19 @@
+# -*- mode: Python -*-
+
+# For more on the `test_go` extension:
+#     https://github.com/tilt-dev/tilt-extensions/tree/master/tests/golang
+# For more on tests in Tilt: https://docs.tilt.dev/tests_in_tilt.html
+load('ext://tests/golang', 'test_go')
+
+# these unit tests are fast/well cached, so it's easy to run them all whenever
+# a file changes and get fast feedback
+# (The `skipintegration` tag prevents this test resource from running our very
+# slow integration test suite)
+test_go('unit-tests', '.', '.',
+        recursive=True, tags=['skipintegration'])
+
+# the integration tests are slow and clunky, so make them available from the sidebar
+# and let the developer run them manually whenever it's necessary
+test_go('integration-tests', './integration', '.',
+        extra_args=["-v"],
+        trigger_mode=TRIGGER_MODE_MANUAL, auto_init=False)
```

Comment:
can you make this auto_init on `tilt ci`, to ensure it doesn't break?

Reply:
ah i had a failing test in here (also in the JS example) to show people what that looks like -- but I can remove that if you think it's better to be able to run these in CI.